app/views/games/_table.html.erb

```diff
@@ -10,20 +10,19 @@
 <% else %>
   <table class="game-table table table-bordered table-striped">
     <tr>
-      <th>Name</th>
-      <% if show_console %>
-        <th>Console</th>
-      <% end %>
+      <th>Top Image</th>
+      <th width='80%'>Name</th>
     </tr>
     <% @matches.each do |game| %>
       <tr>
+        <td>
+          <%= game.pictures.empty? ? 'No image!' : link_to(image_tag(game.top_rated_picture.image_url, class: 'table-image'), game) %>
```

Comment:
I agree! I wasn't sure how you'd feel about throwing a placeholder asset in there so I stuck with the text. It actually inspired me to look into how we might be able to generate images automatically and I whipped up [this little script](https://github.com/XanderStrike/steam-grid-image-generator).

They look just barely workable though, definitely not ready for prime time. I'll add a placeholder image in another commit. Is "No image!" okay for the text in the image?